index.html

```diff
@@ -3554,24 +3554,30 @@ <h5>JWT Encoding</h5>
 in the <a>verifiable credential</a> <a>subject</a>.
               </li>
               <li>
-<code>aud</code> MUST represent the <code>subject</code> <a>property</a> of the
-consumer of the <a>verifiable presentation</a>.
+<code>aud</code> MUST represent (i.e., identify) the intended audience
```

Comment:
This change could be considered substantive, although I expect no implementations would change.